apps/web-giddh/src/app/search/components/sidebar-components/search.sidebar.component.html

```diff
@@ -85,22 +79,17 @@
                     </div>
                 </div>
                 <div>
-                    <label>&nbsp;</label>
                     <button
-                        [disabled]="search.invalid"
-                        class="btn btn-sm btn-success"
+                        [disabled]="!groupUniqueName"
+                        mat-stroked-button
+                        color="primary"
                         (click)="getClosingBalance(false, $event)"
                     >
                         {{ commonLocaleData?.app_search }}
                     </button>
                 </div>
                 <div>
-                    <label>&nbsp;</label>
-                    <button
-                        [disabled]="search.invalid"
-                        class="btn btn-sm btn-default"
-                        (click)="getClosingBalance(true, $event)"
-                    >
+                    <button [disabled]="!groupUniqueName" mat-stroked-button (click)="getClosingBalance(true, $event)">
```

Comment:
add color in mat button

Reply:
Default button "Refresh" will not change color